XpressAI/xircuits
# 📖 Library Component Descriptions Update

# Description

With https://github.com/XpressAI/xircuits/pull/171, we can now have rich component descriptions. This PR adds descriptions to all ~50 of them.

In general, this is the way it's formatted:


```
A short description of the component.

##### Reference: 
- A hyperlink to the reference

##### inPorts:
- inPort Name: short description on what it expects.
    Default: insert default value when needed.

##### outPorts:
- outPort Name: short description on what it outputs
    Default: insert default value when needed.
```


## References

https://github.com/XpressAI/xircuits/pull/171

## Pull Request Type


## Type of Change


# Tests

This would be a proof read PR. You'll need to check on all component descriptions and see if they make sense / formatted correctly. In general the steps would be:

  1. Install Xircuits wheel with the markdown render feature. Try this one https://github.com/XpressAI/xircuits/suites/7537756309/artifacts/310797122
  2. Fetch this PR's components `xircuits-components --branch fahreza/update-desc`
  3. Run Xircuits
  4. Drag each component one by one to the canvas, and check the rendering. 

## Tested on?


# Notes

It's a tad cumbersome task to do, good luck. 😄
